SAK-34075: Remove title truncation for assignments

plukasew opened the pull request:
https://jira.sakaiproject.org/browse/SAK-34075

Sometimes assignment titles are truncated after 40 or 64 characters, so the full title never makes it to the client. This can be problematic for longer assignment titles that have the same prefix.

The truncation that happens in the Velocity templates should be removed. In all cases I can see, the browser will wrap the longer titles just fine. If truncation is discovered to be needed later, it can and should be done with CSS.

jonespm commented:
Did you test this out on a smaller/mobile view with long titles? I agree with using text-overflow: ellipsis; rather than truncating it in the html but we should verify that it does look fine in all cases and doesn't need a css change.

plukasew commented:
Yes, the text wraps as expected in a narrow view. The only issue was on the reorder page, which is why I removed the "title" class that prevents word wrap. This was a problem anyway before I removed the limit, as the text would overlap other text or break out of its container at certain sizes.
